bluetooth_mesh/messages/test/test_scene.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+from construct import SelectError, ValidationError
+
+from bluetooth_mesh.messages.scene import SceneMessageOpcode, SceneMessage
+
+invalid = [
+    pytest.param(
+        SelectError,
+        SceneMessageOpcode.SCENE_RECALL,
+        dict(scene_number=0, tid=30),
+        id="SCENE_RECALL_minimal_with_prohibited_scene_number"),
+    pytest.param(
+        SelectError,
+        SceneMessageOpcode.SCENE_RECALL,
+        dict(scene_number=0, tid=30, transition_time=30000, delay=0.3),
+        id="SCENE_RECALL_with_transition_with_prohibited_scene_number"),
+    pytest.param(
+        SelectError,
+        SceneMessageOpcode.SCENE_RECALL_UNACKNOWLEDGED,
+        dict(scene_number=0, tid=30),
+        id="SCENE_RECALL_UNACKNOWLEDGED_minimal_with_prohibited_scene_number"),
+    pytest.param(
+        SelectError,
+        SceneMessageOpcode.SCENE_RECALL_UNACKNOWLEDGED,
+        dict(scene_number=0, tid=30, transition_time=30000, delay=0.3),
+        id="SCENE_RECALL_UNACKNOWLEDGED_with_transition_with_prohibited_scene_number"),
+    pytest.param(
+        ValidationError,
+        SceneMessageOpcode.SCENE_STORE,
+        dict(scene_number=0),
+        id="SCENE_STORE_with_prohibited_scene_number"),
+    pytest.param(
+        ValidationError,
+        SceneMessageOpcode.SCENE_STORE_UNACKNOWLEDGED,
+        dict(scene_number=0),
+        id="SCENE_STORE_UNACKNOWLEDGED_with_prohibited_scene_number")
+]
+
+valid = [
+    pytest.param(
+        b'\x82\x41',
+        SceneMessageOpcode.SCENE_GET,
+        dict(),
+        id="SCENE_GET"),
+    pytest.param(
+        b'\x82\x42\x01\x00\x1e',
```

Comment:
It would be more readable to use `bytes.fromhex('82 42 01 00 1e')`

Reply:
I agree, but other unit test are already using this format. I don't like doing the same in many ways in one project.

Reply:
Health and config tests do the `fromhex` thing. I'm gonna refactor generics and scene to do this the same way.